readme.md

```diff
@@ -32,7 +32,7 @@
 
 ## Install
 
-```
+```bash
```

Comment:
This is not correct as the below snippet is not valid Bash because of the `$`, but you could use `console`.